cl/search/models.py

```diff
@@ -179,6 +180,7 @@ class Meta:
         verbose_name_plural = "Originating Court Information"
 
 
+@pghistory.track(pghistory.Snapshot())
```

Comment:
We need to ignore the `view_count` here. Can you also check if it's anywhere else that I can't remember? I don't think so, but it'll be annoying to deal with later.

Reply:
yes, no problem, i'll exclude that field